happi/backends/json_db.py

```diff
@@ -49,11 +50,18 @@ class JSONBackend(_Backend):
         Path to JSON file.
     initialze : bool, optional
         Initialize a new empty JSON file to begin filling.
+    cfg_path : str, optional
+        Path to the happi config.
     """
 
-    def __init__(self, path, initialize=False):
+    def __init__(self, path: str, initialize=False, cfg_path=None):
```

Comment:
While we're at it, do you think you could type annotate the rest of these signatures?  (at least the ones you're modifying)  It's something we're trying to be better about, and every little bit helps.